tests/test_api.py

```diff
@@ -506,7 +506,7 @@ def test_linked_thumbnail_url(self):
 
             self.assertEqual(response_version_with_project.get('type'), 'Version')
             self.assertEqual(response_version_with_project.get('id'), self.version['id'])
-            self.assertEqual(response_version_with_project.get('code'), 'Sg unittest version')
+            self.assertEqual(response_version_with_project.get('code'), self.config.version_code)
```

Comment:
The CI sandboxes entities per CI user, so we should be getting the name from the configuration and not a hardcoded value.